backend-exemplar-2018.code-workspace

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I have no idea what effect this file has - I've never seen one around HackO before

Reply:
Ignore this file. Its a VS Code workspace file, which is what I use to edit. I will have to remove it from the repo.